### DISCUSSION TITLE
番剧界面的下载不包括SP，OVA之类的栏目

### DISCUSSION THREAD

**Discussion by linyaoyong** [the1812/Bilibili-Evolved#3127] [功能建议 / Ideas]
番剧界面的分P是分开来的，一个是正片，一个是SP，在同一个页面下，但是下载界面却只能检测到或者是只显示正品的多P下载，OVA或者SP不显示，希望之后能更新功能，增加对SP/OVA那个栏目的下载支持

**Comment by Geeyun-JY3** [comment]
重复：#2834
